iwyu_ast_util.cc

```diff
@@ -1381,6 +1397,16 @@ bool IsDefArgSpecified(unsigned i, const FunctionDecl* func) {
                                               func->getEndLoc());
 }
 
+FunctionDecl* GetRedeclSpecifyingDefArg(unsigned i, FunctionDecl* func) {
+  if (i < func->getNumParams()) {
```

Comment:
I don't really like how you can pass any unsigned value, and the function will silently fall through and return `func`.

I'd rather handle this at the caller side, I'll see if I can suggest something more readable there too.